zebedee-cms/src/main/java/com/github/onsdigital/zebedee/Init.java

```diff
@@ -16,11 +17,13 @@
 import com.github.onsdigital.zebedee.persistence.dao.CollectionHistoryDaoFactory;
 import com.github.onsdigital.zebedee.reader.ZebedeeReader;
 
-import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.error;
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.info;
 
 /**
  * Created by bren on 31/07/15.
  */
+@Priority(2)
```

Comment:
Annotation specifies the order the `Startup` instances should run. The app logging is configured within Reader so we want reader to start before CMS so we don't miss any log events.